Scarb.toml

```diff
@@ -10,16 +10,25 @@ license-file = "LICENSE"
 keywords = ["fixed-point", "wad", "ray", "cairo", "starknet"]
 
 [dependencies]
-starknet = ">= 2.6.0"
+starknet = ">= 2.7.0-rc.3"
+assert_macros = "0.1.0"
 
 [dev-dependencies]
-snforge_std = { git = "https://github.com/foundry-rs/starknet-foundry.git", tag = "v0.19.0" }
+snforge_std = { git = "https://github.com/foundry-rs/starknet-foundry.git", tag = "v0.27.0" }
 
 [lib]
+sierra-text = true
+
 
 [tool.fmt]
 sort-module-level-items = true
 max-line-length = 120
 
 [scripts]
 test = "snforge test"
+
+[profile.fv]
```

Comment:
Will remove this, as it shouldn't be part of the package.

Reply:
In my opinion it would be nice to have it in there, because then we wouldn't need to patch the wadray repo in the generation of the fv basis. Not needing to patch around in Cairo repos the entire reason I added the compiler option to Cairo and removed the `inline(always)` occurrences (that change has been merged today) in corelib. :disappointed:

Reply:
It's just super annoying to tag a branch with the patch along. What's the issue with having in the repo? The profile doesn't get built unless `scarb --profile fv build` is called.

Reply:
But fwiw remove it, it's orthogonal to the version bump anyway

Reply:
Oooh, I see, I didn't realize that about the workflow. Ok, will put it back in.